src/storages/expenses.ts

```diff
@@ -0,0 +1,29 @@
+import { PrismaClient } from '@prisma/client'
+
+export default class ExpensesStorage {
```

Comment:
I was trying to create a reusable class with the correct type of prismaClient.model by passing the key in the constructor but it seems like that is not so easy :)
e.g.
```
const expensesStorage = new Storage(prismaClient, 'expenses');

expensesStorage.findMany() // returns expenses
```

Reply:
Prisma exports a lot of types. Probably you might use them.

But I see storage-like classes with a bit higher level of abstraction. They should call prisma methods and pass arguments to them. findAll can exist but methods like findByEmail also have a place to be.
Or you might not use this abstraction at all and use prismaClient in services.
But approach with services is better.